src/test/java/org/openrewrite/java/migrate/AddScopeToInjectedClassTest.java

```diff
@@ -0,0 +1,195 @@
+/*
+ * Copyright 2023 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.migrate;
+
+import org.openrewrite.DocumentExample;
+import org.openrewrite.test.RewriteTest;
+import org.openrewrite.test.RecipeSpec;
+import org.junit.jupiter.api.Test;
+
+import static org.openrewrite.java.Assertions.java;
+
+class AddScopeToInjectedClassTest implements RewriteTest {
+
+    @DocumentExample
+    @Test
+    void scopeRequired() {
+        rewriteRun(spec -> spec.recipe(new AddScopeToInjectedClass())
+            .cycles(1),
```

Comment:
Maybe best to override `defaults` and define the `spec.recipe(..)` call there, since it's the same everywhere. I don't think we'd need an explicit number of cycles here right?